platforms/android/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
We should replace Keys and other information sensitive information with YOUR-KEY-GOES-HERE